ethereum/remix-project
# Fixed upgrade bug

#### Description
Fixes the issue of error thrown when making an upgrade immediately after deploying a proxy contract.
<img width="592" alt="Screenshot 2023-02-17 at 14 00 55" src="https://user-images.githubusercontent.com/23282059/219659758-cc17b279-ac1f-43cb-9f00-91dd88cd19bc.png">

#### To Test
- Create a new workspace with a UUPS template
- Compile contract
- Select `Deploy with Proxy` and click `Deploy` button
- After deployment Select `Upgrade with Proxy` and click `Deploy` button
- Contract should be deployed and upgrade as expected.
